osu.Game/Screens/OnlinePlay/Multiplayer/Participants/ParticipantPanel.cs

```diff
@@ -201,15 +201,8 @@ protected override void OnRoomUpdated()
             else
                 userModsDisplay.FadeOut(fade_time);
 
-            if (Client.IsHost && !User.Equals(Client.LocalUser))
-                kickButton.FadeIn(fade_time);
-            else
-                kickButton.FadeOut(fade_time);
-
-            if (Room.Host?.Equals(User) == true)
-                crown.FadeIn(fade_time);
-            else
-                crown.FadeOut(fade_time);
+            kickButton.Alpha = Client.IsHost && !User.Equals(Client.LocalUser) ? 1 : 0;
```

Comment:
What happened here?

Reply:
https://github.com/ppy/osu/pull/17314/commits/98b420ee6f2b256aa6e70847751fe311739b2572:

> Since the host is pinned to the top without any animation, it would look
jarring for the crown to fade away from the old panel (and at a 50ms
duration).

(I should have probably included that in the PR description, too late...)

For visual comparsion:
| animation | no animation |
|-----------|--------------|
| ![CleanShot 2022-03-18 at 08 18 37](https://user-images.githubusercontent.com/22781491/158942136-4050dda0-6cae-41e5-bbe8-007a9a6d205c.gif) | ![](https://user-images.githubusercontent.com/22781491/158941376-2abeaafb-d8d5-43f4-86cf-ec2f51e9688d.gif) |

That can alternatively be solved by adding animation to the panel transition, but I couldn't come up with a good animation for it (and the participant list is by itself unanimated so I dunno).

Reply:
Both of these are jarring to me.

Reply:
"No animation" snaps instantly and "animation" snaps near-instantly, the difference is so subtle I had to stare at both for a bit. Could be made smoother with more transition frames/playing the animation slower, I think.

Reply:
Will revisit as a follow-up.